src/style.scss

```diff
@@ -145,12 +171,20 @@ slot {
   &:disabled {
     cursor: not-allowed;
     transform: unset;
-    opacity: .4;
+
+    --auro-color-diabled-button-primary-text-on-dark: #{derive-rgb-values-from-hex($auro-color-text-primary-on-dark)};
```

Comment:
Explain why this is needed? 

There is a postCSS script that runs with the build that outputs fallback CSS values. See the `style.css` file that is created in the `./src` dir.